### PR TITLE
[+] Smart Fields - Add a isFilterable option to let them appear the filters selection.

### PR DESCRIPTION
**Pull request by lucasscariot**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request
- [x] Write changes made in the CHANGELOG.md
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

File: integrations/stripe/setup.js [hunk 1]
```diff
@@ -21,13 +21,13 @@ exports.createCollections = function (Implementation, apimap, collectionAndField
...
+      { field: 'id', type: 'String', isFilterable: false },
+      { field: 'created', type: 'Date', isFilterable: false },
+      { field: 'amount', type: 'Number', isFilterable: false },
+      { field: 'status', type: 'String', isFilterable: false },
+      { field: 'currency', type: 'String', isFilterable: false },
+      { field: 'refunded', type: 'Boolean', isFilterable: false },
+      { field: 'description', type: 'String', isFilterable: false },
       {
         field: 'customer',
         type: 'String',
```
Comment (arnaudbesnier) [inline integrations/stripe/setup.js]: one `isSearchable` is remaining 2 lines below

File: index.js [hunk 2]
```diff
@@ -72,7 +72,7 @@ function requireAllModels(Implementation, modelsDir, displayMessage) {
 function getFields(opts) {
   return _.map(opts.fields, function (field) {
     field.isVirtual = true;
-    field.isSearchable = false;
+    field.isFilterable = true;
```
Comment (arnaudbesnier) [inline index.js]: Nope! it should be customisable as it is for `isSortable`.

File: generators/schemas.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ module.exports = {
           .then(function (schema) {
             integrator.defineFields(model, schema);
             integrator.defineSegments(model, schema);
-            schema.isSearchable = true;
```
Comment (arnaudbesnier) [inline generators/schemas.js]: Nope! I guess this is for the collection, so `isSearchable` is the right key.

**Comment by lucasscariot** [comment]
[(0.5) Hoodline needs the isFilterable option for their Smart Fields.](https://trello.com/c/Vhl9Qg5M/2084-05-hoodline-needs-the-isfilterable-option-for-their-smart-fields)

**Pull request merged**